arbitrum-docs/intro/glossary.mdx

```diff
@@ -5,7 +5,7 @@ user_story: As a user, I want to review a list of terms and definitions that are
 content_type: glossary
 ---
 
-import GlossaryPartial from '../partials/_glossary-partial.md';
+import GlossaryPartial from '../partials/_glossary-partial.mdx';
```

Comment:
This is an autogenerated files in notion-docs/scripts/update.ts so you'll have to modify the filename there as well.

Reply:
Problem fixed in this [PR](https://github.com/OffchainLabs/arbitrum-docs/pull/1456), thank you for pointing this out.